### PR TITLE
Postgres artillery

### PR DESCRIPTION
**Pull request by StevenPurn**
Fixing some tests & cleaning up per the linter. 
New relic & redis are implemented and Postgres is now setup and working.

### CODE REVIEW

File: tests/db.test.js [hunk 1]
```diff
@@ -1,12 +1,12 @@
...
-const db = require('./../db/models/store.js')
+const db = require('./../db/pgresdb.js');
 // Notes:
 // 1. Data must be seeded into Mongodb by running npm run seed.
 // 2. Mongodb must be started by using mongod from terminal
 // 3. use database 'apateez'
 
 
-var place_id = 'ChIJO7u9q5-AhYARiSSXyWv9eJ8'
+var place_id = '23'
```
Comment (pooja-kodavanti) [inline tests/db.test.js]: const?

**Review by pooja-kodavanti — APPROVED** [review]
Looks good, just added one minor comment.

**Comment by pooja-kodavanti** [comment]
I'd also recommend including results.json in your gitignore file.

**Comment by send-on** [comment]
The tests you removed are failing.

**Pull request merged**